apps/src/lib/ui/ValidationStep.jsx

```diff
@@ -50,10 +54,10 @@ export default class ValidationStep extends Component {
           {iconFor(stepStatus)}
           <span>{stepName}</span>
         </div>
-        {(stepStatus === Status.FAILED || displayExplanation) &&
```

Comment:
This line felt a little strange to me, namely that you could set `displayExplanation` to be false, and we would still show the children when `Status.FAILED`. It felt like a cleaner interface to me to just have the ability to say that we always show children, and then if you need different conditions (such as in our case), the consumer can decide to conditionally include children as they see fit.